components/user_settings/modal/user_settings_modal.tsx

```diff
@@ -146,7 +163,8 @@ class UserSettingsModal extends React.Component {
 
     // Called to hide the settings pane when on mobile
     handleCollapse = () => {
-        $(ReactDOM.findDOMNode(this.modalBodyRef.current)).closest('.modal-dialog').removeClass('display--content');
+        const el = ReactDOM.findDOMNode(this.modalBodyRef.current) as any;
```

Comment:
Thank you for removing jQuery :)
I think you can use a real type here as well though, probably something like `HTMLDivElement`?

Reply:
I tried with this:
```js
const el = ReactDOM.findDOMNode(this.modalBodyRef.current) as HTMLDivElement;
el.closest('.modal-dialog').classList.remove('display--content');
```
but currently getting error: `Object is possibly 'null'.` with `el.closest('.modal-dialog')` , do you have any suggestion?

Reply:
You can do a null check on `el` before using it, OR if you know for sure that `el` will exist you can assert it by calling `el!.closest`.